sdk/src/network.cpp

```diff
@@ -378,6 +397,8 @@ int Network::callback_function(struct lws *wsi,
         cout << endl << "Client is sending " << send_buff.func_name() << endl;
 #endif
         std::lock_guard<std::mutex> guard(m_mutex);
+        if (send_buff.DebugString().empty())
```

Comment:
Can't we use `ByteSizeLong()` instead of `DebugString().empty()` to check the size of the message?

Reply:
I tired it but doesn't work all the time. For some messages that have the DebugString() empty the ByteSizeLong() returns values > 0. I've made changes so that it checks whether the command (the function name) is valid or not. This should be enough. Thanks!